core/src/test/scala/unit/kafka/controller/ControllerIntegrationTest.scala

```diff
@@ -291,6 +293,21 @@ class ControllerIntegrationTest extends ZooKeeperTestHarness {
     }, "failed to get expected partition state after entire isr went offline")
   }
 
+  private def preferredReplicaLeaderElection(controllerId: Int, otherBrokerId: Int, tp: TopicAndPartition,
+                                             replicas: Set[Int], leaderEpoch: Int): Unit = {
```

Comment:
Nit: seems like we could pass `otherBroker: KafkaServer` and the code would be simpler all around.

Reply:
done.